content/blogposts/2023/announcing-scip-clang.md

```diff
@@ -0,0 +1,84 @@
+# Announcing scip-clang: a new SCIP indexer for C and C++
+
+[scip-clang](https://github.com/sourcegraph/scip-clang) is our new indexer
+for C and C++ code written from the ground up to natively emit [SCIP](https://github.com/sourcegraph/scip)
+and especially support the wide range of language features present in C++.
+
+scip-clang is now available in beta, and replaces our older indexer lsif-clang.
+We've successfully indexed large codebases such as Chromium with scip-clang.
```

Comment:
If you'd like to try out precise intelligence for C++, you can explore it at [list of repos we want to showcase]

Reply:
Added links for Chromium, LLVM and Postgres.